docs/examples.md

```diff
@@ -23,6 +23,43 @@ Some examples include multiple classes in a single source file. This
 is purely to make the examples easier to follow when browsing on
 GitHub.
 
+## Running and deploying
+
+Each example can be run locally from the command line by changing to
+the example directory and running
+
+```sh
+dotnet run
+```
+
+Alternatively, you can open the solution file in the `examples`
```

Comment:
open the solution file in...

This guide assumes I have a program to open the solution file and know what a solution file (.sln file) is.

Reply:
Yes, it does - because I'd expect every .NET developer to have that information already. I'm very reluctant to reduce the information density by adding extra text that is irrelevant to 99.9% of the expected audience.